testsuite/gateways/wasm/__init__.py

```diff
@@ -65,7 +65,10 @@ def create(self):
         })
 
     def destroy(self):
-        self.mesh.delete_app(self.label, resources="serviceentry")
+        self.mesh.delete_app(self.label, resources="serviceentry,destinationrule")
+        for extension in self.extensions:
+            extension.delete()
```

Comment:
Is this required? Before this commit the cleanup of openshift objects made in extension.py was done correctly as far as I know.

Reply:
Theoretically, no, I just wanted to make sure it gets deleted. I removed it for now and if we see some cases where it doesn't clean up properly I will re-add it